pom.xml

```diff
@@ -35,7 +35,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>git</artifactId>
-            <version>3.10.0</version>
+            <version>3.12.1</version>
```

Comment:
We can start wrapping these version number in descriptive properties name. I have also been guilty of not doing this.

Reply:
I thought we decided that it wasn't worth doing unless we were going to use the version in multiple places?